core/common/src/main/java/alluxio/Constants.java

```diff
@@ -118,7 +118,8 @@
   public static final String KEY_VALUE_WORKER_CLIENT_SERVICE_NAME = "KeyValueWorkerClient";
 
   public static final String REST_API_PREFIX = "v1/api";
-  public static final String LOGGER_TYPE = PropertyKey.LOGGER_TYPE;
+  public static final String LOGGER_TYPE = PropertyKey.LOGGER_TYPE.toString();
```

Comment:
SGTM. Since this change will leave a large footprint, I left a TODO in `Constants.java`